src/CustomerProfile.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from "react";
+
+const CustomerProfile = ({id}) =>{
+    const [profile, setProfile] = useState({})
+    useEffect(() => {
+     fetch(`https://cyf-react.glitch.me/customers/${id}`)
+       .then((res) => res.json())
+       .then((data) => {
+         setProfile(data);
+       });
+   }, [id]);
+    return(
+        id &&(
+        <div>
+    
+            <p>Customer Id: {profile.id}</p>
+            <p>Customer Email: {profile.email}</p>
+            <p>VIP: {profile.vip ? "yes" : "no"}</p>
+            <p>Customer Phone Number: {profile.phoneNumber}</p>
+        
+        </div>
+        ) 
+      
+    )
+   
+}
+
```

Comment:
This component is a little hard to read, I think you would benefit from getting your linting setup with autofix on.
Going to try and schedule some help around this!